packages/block-library/src/latest-posts/edit.js

```diff
@@ -36,6 +39,19 @@ const CATEGORIES_LIST_QUERY = {
 };
 const MAX_POSTS_COLUMNS = 6;
 
+// @todo This should be retrieved dynamically, or from site settings, since it can be changed by the site owner.
+function getImageDimensionsFromSlug( slug = 'thumbnail' ) {
+	switch ( slug ) {
+		case 'large':
+			return [ 1024, 1024 ];
+		case 'medium':
+			return [ 300, 300 ];
+		case 'thumbnail':
+		default:
+			return [ 150, 150 ];
+	}
+}
+
```

Comment:
It's been long enough since I worked on this that I didn't realize this was left in 🤦‍♀

Reply:
I've added the real functionality here, but it required adding a new setting `imageDimensions` to get the mapping of size name to height & width. I also had to pull "full" out of the available options, since we can't actually get a default height & width of all full size images. Maybe a future iteration of this (or of `ImageSizeControl`) could handle this case.